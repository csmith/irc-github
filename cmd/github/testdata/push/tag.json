{
  "ref": "refs/tags/v1.2.0",
  "before": "0000000000000000000000000000000000000000",
  "after": "b0fd4c623c6724406156eaacfc93871e688413af",
  "repository": {
    "id": 243344149,
    "node_id": "MDEwOlJlcG9zaXRvcnkyNDMzNDQxNDk=",
    "name": "irc",
    "full_name": "greboid/irc",
    "private": false,
    "owner": {
      "name": "greboid",
      "email": "greboid@users.noreply.github.com",
      "login": "greboid",
      "id": 189618,
      "node_id": "MDQ6VXNlcjE4OTYxOA==",
      "avatar_url": "https://avatars0.githubusercontent.com/u/189618?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/greboid",
      "html_url": "https://github.com/greboid",
      "followers_url": "https://api.github.com/users/greboid/followers",
      "following_url": "https://api.github.com/users/greboid/following{/other_user}",
      "gists_url": "https://api.github.com/users/greboid/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/greboid/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/greboid/subscriptions",
      "organizations_url": "https://api.github.com/users/greboid/orgs",
      "repos_url": "https://api.github.com/users/greboid/repos",
      "events_url": "https://api.github.com/users/greboid/events{/privacy}",
      "received_events_url": "https://api.github.com/users/greboid/received_events",
      "type": "User",
      "site_admin": false
    },
    "html_url": "https://github.com/greboid/irc",
    "description": "Plugin based IRC library for Go",
    "fork": false,
    "url": "https://github.com/greboid/irc",
    "forks_url": "https://api.github.com/repos/greboid/irc/forks",
    "keys_url": "https://api.github.com/repos/greboid/irc/keys{/key_id}",
    "collaborators_url": "https://api.github.com/repos/greboid/irc/collaborators{/collaborator}",
    "teams_url": "https://api.github.com/repos/greboid/irc/teams",
    "hooks_url": "https://api.github.com/repos/greboid/irc/hooks",
    "issue_events_url": "https://api.github.com/repos/greboid/irc/issues/events{/number}",
    "events_url": "https://api.github.com/repos/greboid/irc/events",
    "assignees_url": "https://api.github.com/repos/greboid/irc/assignees{/user}",
    "branches_url": "https://api.github.com/repos/greboid/irc/branches{/branch}",
    "tags_url": "https://api.github.com/repos/greboid/irc/tags",
    "blobs_url": "https://api.github.com/repos/greboid/irc/git/blobs{/sha}",
    "git_tags_url": "https://api.github.com/repos/greboid/irc/git/tags{/sha}",
    "git_refs_url": "https://api.github.com/repos/greboid/irc/git/refs{/sha}",
    "trees_url": "https://api.github.com/repos/greboid/irc/git/trees{/sha}",
    "statuses_url": "https://api.github.com/repos/greboid/irc/statuses/{sha}",
    "languages_url": "https://api.github.com/repos/greboid/irc/languages",
    "stargazers_url": "https://api.github.com/repos/greboid/irc/stargazers",
    "contributors_url": "https://api.github.com/repos/greboid/irc/contributors",
    "subscribers_url": "https://api.github.com/repos/greboid/irc/subscribers",
    "subscription_url": "https://api.github.com/repos/greboid/irc/subscription",
    "commits_url": "https://api.github.com/repos/greboid/irc/commits{/sha}",
    "git_commits_url": "https://api.github.com/repos/greboid/irc/git/commits{/sha}",
    "comments_url": "https://api.github.com/repos/greboid/irc/comments{/number}",
    "issue_comment_url": "https://api.github.com/repos/greboid/irc/issues/comments{/number}",
    "contents_url": "https://api.github.com/repos/greboid/irc/contents/{+path}",
    "compare_url": "https://api.github.com/repos/greboid/irc/compare/{base}...{head}",
    "merges_url": "https://api.github.com/repos/greboid/irc/merges",
    "archive_url": "https://api.github.com/repos/greboid/irc/{archive_format}{/ref}",
    "downloads_url": "https://api.github.com/repos/greboid/irc/downloads",
    "issues_url": "https://api.github.com/repos/greboid/irc/issues{/number}",
    "pulls_url": "https://api.github.com/repos/greboid/irc/pulls{/number}",
    "milestones_url": "https://api.github.com/repos/greboid/irc/milestones{/number}",
    "notifications_url": "https://api.github.com/repos/greboid/irc/notifications{?since,all,participating}",
    "labels_url": "https://api.github.com/repos/greboid/irc/labels{/name}",
    "releases_url": "https://api.github.com/repos/greboid/irc/releases{/id}",
    "deployments_url": "https://api.github.com/repos/greboid/irc/deployments",
    "created_at": 1582744177,
    "updated_at": "2020-03-21T16:46:59Z",
    "pushed_at": 1584809651,
    "git_url": "git://github.com/greboid/irc.git",
    "ssh_url": "git@github.com:greboid/irc.git",
    "clone_url": "https://github.com/greboid/irc.git",
    "svn_url": "https://github.com/greboid/irc",
    "homepage": "",
    "size": 184,
    "stargazers_count": 0,
    "watchers_count": 0,
    "language": "Go",
    "has_issues": true,
    "has_projects": true,
    "has_downloads": true,
    "has_wiki": true,
    "has_pages": false,
    "forks_count": 1,
    "mirror_url": null,
    "archived": false,
    "disabled": false,
    "open_issues_count": 0,
    "license": {
      "key": "mit",
      "name": "MIT License",
      "spdx_id": "MIT",
      "url": "https://api.github.com/licenses/mit",
      "node_id": "MDc6TGljZW5zZTEz"
    },
    "forks": 1,
    "open_issues": 0,
    "watchers": 0,
    "default_branch": "master",
    "stargazers": 0,
    "master_branch": "master"
  },
  "pusher": {
    "name": "greboid",
    "email": "greboid@users.noreply.github.com"
  },
  "sender": {
    "login": "greboid",
    "id": 189618,
    "node_id": "MDQ6VXNlcjE4OTYxOA==",
    "avatar_url": "https://avatars0.githubusercontent.com/u/189618?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/greboid",
    "html_url": "https://github.com/greboid",
    "followers_url": "https://api.github.com/users/greboid/followers",
    "following_url": "https://api.github.com/users/greboid/following{/other_user}",
    "gists_url": "https://api.github.com/users/greboid/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/greboid/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/greboid/subscriptions",
    "organizations_url": "https://api.github.com/users/greboid/orgs",
    "repos_url": "https://api.github.com/users/greboid/repos",
    "events_url": "https://api.github.com/users/greboid/events{/privacy}",
    "received_events_url": "https://api.github.com/users/greboid/received_events",
    "type": "User",
    "site_admin": false
  },
  "created": true,
  "deleted": false,
  "forced": false,
  "base_ref": "refs/heads/master",
  "compare": "https://github.com/greboid/irc/compare/v1.2.0",
  "commits": [

  ],
  "head_commit": {
    "id": "b0fd4c623c6724406156eaacfc93871e688413af",
    "tree_id": "13e7077b2f5bcd7252fc357c36fc918a14d52d73",
    "distinct": true,
    "message": "Use the right name",
    "timestamp": "2020-03-21T16:46:52Z",
    "url": "https://github.com/greboid/irc/commit/b0fd4c623c6724406156eaacfc93871e688413af",
    "author": {
      "name": "Greg Holmes",
      "email": "git@greg.holmes.name",
      "username": "greboid"
    },
    "committer": {
      "name": "Greg Holmes",
      "email": "git@greg.holmes.name",
      "username": "greboid"
    },
    "added": [

    ],
    "removed": [

    ],
    "modified": [
      "Dockerfile.github"
    ]
  }
}