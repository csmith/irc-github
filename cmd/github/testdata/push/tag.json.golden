[greboid/irc] greboid created tag v1.2.0 from branch master - https://github.com/greboid/irc/compare/v1.2.0